windows/ReactTestApp/MainPage.h

```diff
@@ -1,5 +1,7 @@
 #pragma once
 
+#include <any>
```

Comment:
We should always `#include` what we use:
```suggestion
#include <any>
#include <optional>
#include <map>
#include <string>
```